### ISSUE TITLE
Finish range proof verification

### ISSUE THREAD

**Issue by rkuris** [ava-labs/firewood#738]
Already started in https://github.com/ava-labs/firewood/pull/693 but needs completion

**Comment by rkuris** [comment]
Rebased, warning cleanups next

**Comment by demosdemon** [comment]
note to self: create separate issues for sub-prs. this auto-close is annoying and I wasn't able to figure out how to stop it for some PRs

**Comment by rkuris** [comment]
There are still a bunch of tests marked disabled because of this ticket. Reopened so we can track removing them.

**Comment by demosdemon** [comment]
> There are still a bunch of tests marked disabled because of this ticket. Reopened so we can track removing them.

yep, I mistook this for a different task. I realized it late last night. Thanks for re-opening!
